components/Author/Tabs/FeedCard.tsx

```diff
@@ -541,6 +558,26 @@ const styles = StyleSheet.create({
     fontSize: 14,
     marginRight: 5,
   },
+  tag: {
+    display: "inline-block",
+    backgroundColor: "#E8EEFD",
+    color: "#4F53F6",
+    fontWeight: 600,  
+    padding: "6px 20px",
+    borderRadius: "18px",
+    margin: "0px 10px 10px 0px",
```

Comment:
Maybe you already given it some thought and this is already addressed here. Wondering if there will be a bunch of empty space in the case a feed card has no tags